### DISCUSSION TITLE
font awesome icon in mermaid diagram doesn't work

### DISCUSSION THREAD

**Discussion by azzamsa** [usememos/memos#3568] [Q&A]
Hi,

The font awesome icon in mermaid diagram doesn't show up.

![image](https://github.com/usememos/memos/assets/17734314/356ab403-ff90-4d5f-91cd-4f2c9eb2b414)

![image](https://github.com/usememos/memos/assets/17734314/6df19a88-f383-4b89-a53e-7f0cecce2fb6)

Version: v0.22.2

Thank you for Memos 🍜

**Comment by Clusters** [comment]
You have to install the font-awesome icon pack, which is not installed by default.
Follow up this discussion #4429 on how to do that.
